queries/r/highlights.scm

```diff
@@ -10,7 +10,7 @@
 (string) @string
 (string (escape_sequence) @string.escape)
 
-(comment) @comment
+(comment) @comment @spell
```

Comment:
Maybe also `@string`? Or do strings contain mostly non-text data

Reply:
They contain mainly text data, but I wouldn't recommend it, since strings are used for subsetting and thus camel case as well as other common variable names will have wrong spelling. 

![image](https://user-images.githubusercontent.com/26463953/194411205-6b4dd90d-5552-43a4-ba44-300f46530e25.png)